pkg/controllers/cisnetworkpolicy/controller.go

```diff
@@ -0,0 +1,166 @@
+package cisnetworkpolicy
+
+import (
+	"context"
+	"net"
+	"sort"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/rancher/k3s/pkg/server"
+	coreclient "github.com/rancher/wrangler-api/pkg/generated/controllers/core/v1"
+	core "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	FlannelPresenceLabel         = "flannel.alpha.coreos.com/public-ip"
+	FlannelHostNetworkPolicyName = "rke2-flannel-host-networking"
+)
+
+func CISNetworkPolicyController(ctx context.Context, sc *server.Context) error {
+	Register(ctx, sc.Core.Core().V1().Node(), sc.K8s)
+
+	return nil
+}
+
+func Register(ctx context.Context,
```

Comment:
exporting this seems unnecessary (granted we break api compatibility at will with no regards to semver whatsoever)

Reply:
No longer exported